torch/onnx/symbolic_opset9.py

```diff
@@ -931,6 +931,15 @@ def selu(g, input):
 
 @parse_args('v', 'i', 'v')
 def index_select(g, self, dim, index):
+    index_const = sym_help._maybe_get_scalar(index)
+    if not sym_help._is_value(index_const):
+        index = g.op("Constant", value_t=torch.LongTensor([index_const]))
+    elif index.type().kind() == "CompleteTensorType" or index.type().kind() == "DimensionedTensorType":
+        if index.type().dim() == 0:
```

Comment:
dim() != 1?

Reply:
we want to work with scalar case which is dim() == 0.